client/src/components/App.js

```diff
@@ -30,7 +31,7 @@ const App = (props) => {
       <TopBar user={currentUser} />
       <Switch>
         <Route exact path="/">
-          <h2>Hello from react</h2>
+          <PlanetList />
         </Route>
```

Comment:
consistency: lets choose one way to render this route, unless you are passing special props down